packages/angular_devkit/core/src/json/parser.ts

```diff
@@ -860,3 +879,23 @@ export function parseJson(input: string, mode = JsonParseMode.Default): JsonValu
 
   return parseJsonAst(input, mode).value;
 }
+
+/**
+ * Parse a JSON string into its value.  This discards the AST and only returns the value itself.
+ * It also absorbs JSON parsing errors and return a new error with the path in it. Useful for
+ * showing errors when parsing from a file.
+ * @param input The string to parse.
+ * @param mode The mode to parse the input with. {@see JsonParseMode}.
+ * @returns {JsonValue} The value represented by the JSON string.
+ */
+export function parseJsonFile(input: string, mode = JsonParseMode.Default, path: string) {
```

Comment:
based on the name of the function, I'd expect `input` to be a filename and the function to load and parse an actual file.  Can `parseJson` include a `path`/`sourceFile` option (bike shedding the name, sorry)?

Reply:
I was thinking about that. I'll add it, it's no big deal.